### ISSUE TITLE
Then...and not indented correctly

### ISSUE THREAD

**Issue by darkdragon-001** [alexkrechik/VSCucumberAutoComplete#378]
**Describe the bug**
`Then`...`and` is not indented correctly.

**To Reproduce**
Use this file and run _Format document_:
```gherkin
Feature: feature

  Scenario: S1
    Then a
  and b

  Scenario: S2
    Then a
  and b

  Scenario: S3
    Then a
and b
```

**Expected behavior**
`and` clause should have the same indent as `Then` clause.

**Cucumberautocomplete part of VSCode settings:**
Using defaults:
```
{}
```

**Note** It works when explicitly setting `"and": 2`:
```
{
    "cucumberautocomplete.formatConfOverride": {
        "and": 2
    }
}
```

**Step definition:**
Not needed.

**Gherkin step line**
See above.

**Comment by lacell75** [comment]
I have the same bug in my side. Could you fix it?

**Comment by alexkrechik** [comment]
Should be fixed after 2.16.0 release using 'relativeUp' option when required.

**Comment by alexkrechik** [comment]
```
  "cucumberautocomplete.formatConfOverride": {
    "Then": 5,
    "And": "relativeUp",
  }
```
Would be resulted in:
<img width="314" alt="Screenshot 2024-05-15 at 21 49 52" src="https://github.com/alexkrechik/VSCucumberAutoComplete/assets/5772406/3b23a7d6-a999-4d5b-838f-33435cf0bd92">

**Comment by alexkrechik** [comment]
This issue is to be fixed or implemented in the 3.0.0 version of the extension. If this issue is unresolved or there are additional notes, please create a new ticket with the necessary details.
